scripts/docker/Dockerfile.openmptarget

```diff
@@ -41,7 +41,7 @@ RUN CMAKE_URL=https://github.com/Kitware/CMake/releases/download/v${CMAKE_VERSIO
     rm ${CMAKE_SCRIPT}
 ENV PATH=${CMAKE_DIR}/bin:$PATH
 
-ARG LLVM_VERSION=llvmorg-17.0.3
+ARG LLVM_VERSION=llvmorg-20.1.3
```

Comment:
Is there a chance that we don't have to build the compiler ourselves? https://github.com/kokkos/kokkos/pull/8132 uses the package manager with https://apt.llvm.org/bookworm/dists/.

Reply:
I do not have any strong objection to either method. If the consensus is to use package manager, I can do it that way.